src/app.js

```diff
@@ -1 +1,52 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+console.log("Hallo daar");
+
+const countryCurrency = (currencies) => {
+    let localC = "";
+    for (let i = 0; i < currencies.length; i++) {
+        if (i > 0) {
+            localC += ' and ' + currencies[i].name + 's. ';
+        } else {
+            localC = currencies[i].name;
+        }
+    }
+    console.log(localC);
+    return localC
+}
+
+
+async function getCountryInformation() {
+    const BASE = "https://restcountries.com/";
+    const ENDPOINT = "v2/name/";
```

Comment:
Variabelen camelCase schrijven
```suggestion
    const base = "https://restcountries.com/";
    const endpoint = "v2/name/";
```